.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.12
```

Comment:
Why not use 1.14?

Reply:
I'll upgrade in another PR. This one is about migrating to GitHub Actions and making sure that everything works as with Travis CI.